lib/cssom.js

```diff
@@ -11,12 +11,26 @@
 declare class StyleSheet {
   disabled: boolean;
   href: string;
+  media: MediaList;
   ownerNode: Node;
   parentStyleSheet: ?StyleSheet;
   title: string;
   type: string;
 }
 
+declare class StyleSheetList {
+  length: number;
+  [index: number]: StyleSheet;
+}
+
+declare class MediaList {
+  mediaText: DOMString;
+  length: number;
+  item(index: number): DOMString;
```

Comment:
if `index` doesn't exist, this returns `null` so this should be `string | null` (not `?string` since I don't think there's any way for it to return undefined)

also, let's add an `[index: number]: string` here too, I think array lookups are supported:

pick any page with CSS (say, this one) and try this in the console:
```
document.querySelector("link[rel=stylesheet]").media = "screen"; document.styleSheets[0].media[0]
```

also supported by [cssom](http://dev.w3.org/csswg/cssom/#the-medialist-interface) and this kinda ambiguous line from [DOM](http://www.w3.org/TR/DOM-Level-2-Style/stylesheets.html#StyleSheets-MediaList) -- "The items in the MediaList are accessible via an integral index, starting from 0."